### DISCUSSION TITLE
Property 'crud' does not exist on type 'ObjectDefinitionBlock<"Mutation">'

### DISCUSSION THREAD

**Discussion by TheoMer** [prisma/prisma#3346] [Q&A] [answered]
"@nexus/schema": "^0.15.0",
"nexus-plugin-prisma": "^0.16.1",
"ts-node": "^8.10.2",
"typescript": "^3.9.6",
"ts-node-dev": "^1.0.0-pre.50"

So, on both my mutationType() and queryType() I'm getting the above mentioned error in VSCode. My code is as follows:

```ts
import { arg, idArg, stringArg, intArg, queryType, mutationType} from '@nexus/schema';

export const Mutation = mutationType({
    definition(t) {
  
      t.crud.createOneItem({
```

What's the issue here? My schema.ts is as follows:

```ts
import * as path from 'path';
import * as types from './types';
import { join } from 'path'

import { makeSchema } from '@nexus/schema';
import { nexusSchemaPrisma } from 'nexus-plugin-prisma/schema';

const nexusPrisma = nexusSchemaPrisma({
    experimentalCRUD: true,
    paginationStrategy: 'prisma',
    //prismaClient: (ctx: Context) => ctx.prisma,
  })

export const schema = makeSchema({
  types,
  //plugins: [nexusSchemaPrisma()],
  plugins: [nexusPrisma],
  outputs: {
    schema: path.join(__dirname, './../schema.graphql'),
    typegen: path.join(__dirname, './generated/nexus.ts'),
  },
  /*typegenAutoConfig: {
    sources: [
      {
        source: '@prisma/client',
        alias: 'prisma',
      },
      {
        source: join(__dirname, 'types.ts'),
        alias: 'ctx',
      },
    ],
    contextType: 'ctx.Context',
  },*/
});
```

**Comment by ryands17** [comment, marked as answer]
Hey @TheoMer 👋 
The code looks correct. Could you try restarting the server and check again. I have a similar example [here](https://github.com/ryands17/nexus-auth/) that you can take reference from.

If that still doesn't work, you would need to send me a reproduction of what you have created currently so that I could check.

**Reply by TheoMer** [reply]
Hey @ryands17 I've created a repo here: https://github.com/TheoMer/Prisma2-Miration/

**Reply by ryands17** [reply]
I'm not facing any such error in VSCode
![image](https://user-images.githubusercontent.com/19697099/90633818-5fbb8600-e244-11ea-8ef0-8a4f7bfd21b4.png)

Could you double-check by restaarting the server?

**Reply by TheoMer** [reply]
Hey @ryands17 it appears windows is giving me the runaround. A reinstall did the trick.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
